app/helpers/application_helper.rb

```diff
@@ -145,4 +145,26 @@ def concurrent_events(event)
     end
     concurrent_events
   end
+
+  def event_types(conference)
+    all = conference.program.event_types.map { |et| et.title.pluralize }
+    first = all[0...-1]
+    last = all[-1]
+    ets = ''
+    if all.length > 1
+      ets << first.join(', ')
+      ets << " and #{last}"
+    else
+      ets = all.join
+    end
+    ets
+  end
```

Comment:
I don't see a problem with that additional comma. Let's just use plain `to_sentence` and keep it simple.